### PR TITLE
Fix typos in docs

### PR DESCRIPTION
**Pull request by chaitanyamogal**
Fixed some typos and grammatical mistakes in docs. @ReimarBauer

### CODE REVIEW

File: docs/01_about.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ scientists in the field of atmospheric science. The purpose is to have a
 tool that simplifies the process for planning a scientific flight in
 which parameters of the atmosphere are measured. The research aircrafts
 typically carry a comprehensive scientific payload comprised of data
-aquisition instruments by different companies and research institutions.
+acquisition instruments by different companies and research institutions.
```
Comment (ReimarBauer) [inline docs/01_about.md:12]: deepl.com knows both words.

(which word is better, @joernu76 ?)

File: docs/02_installation.md [hunk 2]
```diff
@@ -70,24 +70,24 @@ The configuration is described in the section
 [mss-configuration](https://mss.readthedocs.io/en/stable/usage.html#mss-configuration) 
 
 ### mswms server
-To tryout the setup you can use demodata. Read about a [server based installation](https://mss.readthedocs.io/en/stable/deployment.html). 
+To try out the setup you can use demo data. Read about a [server based installation](https://mss.readthedocs.io/en/stable/deployment.html). 
    
     (mssenv) $ mswms_demodata --create
     (mssenv) $ mswms
 
-This data is than available on localhost:8081.
```
Comment (ReimarBauer) [inline docs/02_installation.md:78]: :)

**Review by ReimarBauer — COMMENTED** [review]
Thx, there are a lot of typos, than/then mistakes. For the scientific part @joernu76 has the last word :)

File: docs/01_about.md (hunk 1 above)
Comment (joernu76) [reply docs/01_about.md:12]: With "c" is correct. Without "c" is a common mis-spelling and deepl is trained by the internet, isn't it? Not the best reference for spelling I'd say.

**Review by joernu76 — APPROVED** [review]
Oh my, that was a lot of  mistakes.

**Pull request merged**